types.d.ts

```diff
@@ -1093,4 +1093,3 @@ declare module "@asyncapi/parser" {
      */
     function registerSchemaParser(parserModule: any): void;
 }
-
```

Comment:
Not sure if the auto generator will have a problem with this new line. Think you need to remove it.

Reply:
dist and types is generated on every release, so bot will update it :P

Reply:
fixed :P